forge/forge-policy.json

```diff
@@ -84,15 +84,6 @@
         }
       }
     },
-    {
-      "Sid": "ECSDescribe",
-      "Effect": "Allow",
-      "Action": [
-        "ecs:DescribeTasks",
-        "ecs:DescribeContainerInstances"
-      ],
-      "Resource": "*"
-    },
```

Comment:
These are needed by nextflow https://github.com/nextflow-io/nextflow/blob/a30dbbbcd0e0eb6334a3a4b70dd9e35d1cc85340/plugins/nf-amazon/src/main/nextflow/cloud/aws/batch/AwsBatchHelper.groovy#L136-L143

Reply:
That may be correct for Forge setup (it should be tested and check the nextflow logs in any case, side effect could be silently breaking task metadata collection). 

But surely it should for the policy for "manual" CE config